src/workspace-data/ImportStatus.ts

```diff
@@ -19,29 +19,45 @@ const ImportStatus = () => {
           key: job.jobId,
           job,
           onDone: () => {
-            asyncImportJobStore.update(_.reject({ jobId: job.jobId }));
+            asyncImportJobStore.update(_.reject({ jobId: job.jobId, targetWorkspace: job.targetWorkspace }));
           },
         }),
       _.uniq(jobs)
     )
   );
 };
 
-function ImportStatusItem({ job: { targetWorkspace, jobId }, onDone }) {
+interface ImportStatusItemProps {
+  job: {
+    targetWorkspace: { namespace: string; name: string };
+    jobId: string;
+    wdsProxyUrl?: string;
+  };
```

Comment:
Could this use the existing AsyncImportJob type?

```suggestion
  job: AsyncImportJob;
```

Reply:
Hmm, it gets mad because wdsProxyUrl may or may not be present.  Fumbling with typescript but if you know off the top of your head how to deal with that I could do as you ask

Reply:
Instead of always destructuring `wdsProxyUrl` out of the job, might have to change it to something like:

```ts
const wdsProxyUrl = 'wdsProxyUrl' in job ? job.wdsProxyUrl : undefined;
```

Reply:
Alright I gave it a try in [19b5f7c59](https://github.com/DataBiosphere/terra-ui/pull/4496/commits/19b5f7c5990207ca83442f7b67c81f5f38193238)